test-packages/basic-app/test/simple-test.js

```diff
@@ -108,6 +108,18 @@ describe('simple acceptance', function() {
     expect(response.body).to.contain("FastBoot compatible vendor file: FastBoot default default value");
   });
 
+  it('the body is contained between markers', async () => {
```

Comment:
Move this test to the simple-test which I think is a better location for this test.